### ISSUE TITLE
Allow user to edit if has the rainlab.pages.manage_content permission

### ISSUE THREAD

**Issue by mariavilaro** [rainlab/editable-plugin#34]
I have an user without cms.manage_content permission, but with rainlab.pages.manage_content enabled. I managed to solve it with this change: 

```
   public function checkEditor()
    {
        $backendUser = BackendAuth::getUser();
        return $backendUser && ($backendUser->hasAccess('cms.manage_content') || $backendUser->hasAccess('rainlab.pages.manage_content'));
    }
```

Sorry don't have the time to do a pull request now, in nobody takes care I will do it when I can!

**Comment by daftspunk** [comment]
Yes this looks great. Someone should submit a PR for this and it will be accepted. I am unable to do it right now also. Cheers.

**Comment by LukeTowers** [comment]
Fixed by #35
